src/Data/Swagger/Internal.hs

```diff
@@ -1015,10 +1027,8 @@ instance ToJSON (CollectionFormat t) where
   toJSON CollectionMulti = "multi"
 
 instance ToJSON (ParamSchema t) where
-  toJSON = omitEmptiesExcept f . genericToJSONWithSub "items" (jsonPrefix "paramSchema")
-    where
-      f "items" (Array _) = True
```

Comment:
it falls out naturally as fields from sub-object aren't omitted. OTOH i'm looking for some kind of symmetric variant to do `FromJSON`, e.g. only omit fields which are specified in `defaults` if the values match.